app/assets/stylesheets/app/user_tags.scss

```diff
@@ -0,0 +1,123 @@
+.modal-overlay {
+  position: fixed;
+  z-index: 1000;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, .5);
+  display: table;
+  transition: opacity .3s ease;
+}
+
+.modal-container {
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  width: 500px;
+  background-color: #fff;
+  border-radius: 2px;
+  box-shadow: 0 2px 8px rgba(0, 0, 0, .33);
+  transition: all .3s ease;
+  -webkit-transform: translate(-50%, -50%);
+  -moz-transform: translate(-50%, -50%);
+  transform: translate(-50%, -50%);
+
+  .close {
+    float: right;
+    padding: 2px 5px;
+    height: 30px;
+    width: 40px;
+    background: #2fcab5;
+    border-color: #2fcab5;
+    border-style: groove;
+    cursor: pointer;
+  }
+}
+
+.gray-border {
+  border: solid;
+  border-width: thin;
+  border-color: rgb(172, 171, 171);
+}
+
+.modal-body {
+  padding: 27px 27px 8px;
+
+  .dropdown-container {
+    padding-bottom: 10px;
+
+    .pl-dropdown.select {
+      width: 80%;
+      height: 80%;
+    }
+
+    .select__btn {
+      width: 100%;
+    }
+  }
+
+  .modal-tags {
+    width: 445px;
+    height: 200px;
+  }
+
+}
+
+.modal-leave-active {
+  opacity: 0;
+}
+
+.modal-footer {
+  margin-top: 10px;
+  display: flex;
+  justify-content: flex-end;
+
+  .btn-addtags {
+    border-style: hidden;
+    text-align: center;
+    text-decoration: none;
+    padding: 6px 40px;
+    font-size: 13px;
+    cursor: pointer;
+    border-radius: 15px;
+    background-color: #2fcab5;
+    color: rgb(255, 255, 255);
+  }
+}
+
+.show-tags {
+  margin-top: 10px;
+  .profile-tag {
+    display: inline-block;
+
+    .no-tags {
+      padding-left: 30px;
+      padding-right: 30px;
+      border-style: dashed;
+      border-color: rgb(181, 181, 181);
+      color: rgb(157, 154, 154);
+      border-width: thin;
+    }
+
+    .btn-modal {
+      position: relative;
+      width: 25px;
+      height: 25px;
+      border: none;
```

Comment:
**[stylelint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected value "none" for property "border"   declaration-property-value-blacklist